electrolytes/solvation/solvate.py

```diff
@@ -353,7 +352,8 @@ def dump_xyzs(
     # TODO: CHANGE THESE
     # `xyz_dir` should point to a root-level directory with *.xyz files
     # *.xyz's can be nested multiple levels down - this script will recursively search for them
-    xyz_dir = Path("xyzs")
+    # xyz_dir = Path("xyzs")
+    xyz_dir = Path("/home/ewcss/data/omol24/20240607_solvation/test")
```

Comment:
If this is the output directory, can it be made a commandline flag of the launch script via an argparse?

Reply:
Yeah, that's fair. I was lazy and didn't want to write an argument parser.

Reply:
Will do that, hopefully soon.